lab_1/main.py

```diff
@@ -87,25 +149,53 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
+    i = 0
+    k = 0
+    for i in range(len(tokens)):  # проверка строки на корректность
```

Comment:
use checks with isinstance here too

Reply:
I will, but please tell me how to fix the test errors. This is killing me. It worked well on PyCharm, with only ('tokenize.txt') in TokenizeTest instead of (lab_1/tokenize.txt), but what is wrong now?